### PR TITLE
website/docs: expand beta beta / install docs

### PR DESCRIPTION
**Pull request by BeryJu**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [x] The documentation has been updated
-   [x] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/installation/beta.mdx [hunk 1]
```diff
@@ -16,7 +16,8 @@ import TabItem from "@theme/TabItem";
   values={[
     {label: 'Docker Compose', value: 'docker-compose'},
     {label: 'Kubernetes', value: 'kubernetes'},
-  ]}>
+  ]}
+  groupId="platform">
```
Comment (BeryJu) [inline website/docs/installation/beta.mdx:20]: This makes both tabs sync their state

File: website/docs/installation/docker-compose.md [hunk 2]
```diff
@@ -92,27 +92,8 @@ docker-compose up -d
...
 The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file. Each time you upgrade to a newer version of authentik, you download a new `docker-compose.yml` file, which points to the latest available version. For more information, refer to the **Upgrading** section in the [Release Notes](../releases).
 
-To start the initial setup, navigate to `https://<your server's IP or hostname>:9000/if/flow/initial-setup/`.
+To start the initial setup, navigate to `http://<your server's IP or hostname>:9000/if/flow/initial-setup/`.
 
 There you are prompted to set a password for the akadmin user (the default user).
 
-## Explanation
-
-:::warning
```
Comment (BeryJu) [inline website/docs/installation/docker-compose.md:101]: This was duplicated here due to the #6250 PR

File: website/docs/releases/2023/v2023.6.md [hunk 3]
```diff
@@ -36,12 +36,11 @@ The `-O` flag retains the downloaded file's name, overwriting any existing local
 
 ### Kubernetes
 
-Update your values to use the new images:
+Upgrade the helm chart to the new version
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.6.md:39]:
```suggestion
Upgrade the Helm Chart to the new version using the following commands:
```

File: website/docs/releases/_template.md [hunk 4]
```diff
@@ -24,12 +24,11 @@ The `-O` flag retains the downloaded file's name, overwriting any existing local
 
 ### Kubernetes
 
-Update your values to use the new images:
+Upgrade the helm chart to the new version
```
Comment (tanberry) [inline website/docs/releases/_template.md:27]:
```suggestion
Upgrade the Helm Chart to the new version, using the following commands:
```

**Review by tanberry — APPROVED** [review]

**Pull request merged**